notifications_utils/logging.py

```diff
@@ -186,5 +186,5 @@ def process_log_record(self, log_record):
         try:
             log_record["message"] = log_record["message"].format(**log_record)
         except (KeyError, IndexError) as e:
-            logger.exception(f"failed to format log message: {e} not found")
+            logger.warning(f"failed to format log message: {e} not found")
```

Comment:
Actually ... no, because that could theoretically recurse infinitely. The log message getting emitted will contain the broken log message and that would go through this `process_log_record` again.